### PR TITLE
LPS-130156 Create the custom view backend service

### PR DESCRIPTION
**Pull request by izaera**
This is the definition of the custom view entity.

Haven't tested anything yet because it's not being used yet :shrug: .

### CODE REVIEW

**Comment by izaera** [comment]
Please read the commit messages for rationale supporting each decision.

**Comment by izaera** [comment]
ci:test:sf

**Comment by izaera** [comment]
ci:test:relevant

**Comment by izaera** [comment]
@dsanz @nhpatt Can we forward this to Brian? :thinking:

File: modules/apps/frontend-view-state/frontend-view-state-service/service.xml [hunk 1]
```diff
@@ -37,6 +37,39 @@
...
+		<!-- Finder methods -->
+
+		<finder name="U_EI_N" return-type="FVSCustomEntry" unique="true">
+			<finder-column name="userId" />
+			<finder-column name="entityId" />
+			<finder-column name="name" />
+		</finder>
+
+		<!-- References -->
+
```
Comment (dsanz) [inline modules/apps/frontend-view-state/frontend-view-state-service/service.xml:72]: I guess we can add some uniqueness constraints later on right? I'm thinking on making the custom view names a bit more unique, either by creating an unique index on (`name`, `userid`, `entityId`) or even on (`name`, `entityId`)

Comment (izaera) [reply modules/apps/frontend-view-state/frontend-view-state-service/service.xml:72]: We can add them later or in this PR if we already know it...

**Review by nhpatt — APPROVED** [review]
LGTM :)

**Comment by izaera** [comment]
ci:forward

**Pull request closed without merging**